src/mpid/ch4/src/ch4r_win.h

```diff
@@ -15,6 +15,7 @@
 #include "ch4i_util.h"
 #include <opa_primitives.h>
 #include "mpir_info.h"
+#include "ch4r_symheap.h"
```

Comment:
same as comment in the other pr, why need adding this line here in this commit?

Reply:
It is needed for `inline=no` mode.  

We do not get compiling problem when compiling the master branch because this header file is included in `mpidch4r.h`, and `mpidch4r.h` are included in each netmod, and the symmheap routines are called only through the following routine:
```NM_win_allocate_* -> (1) CH4R_win_allocate_* -> (2) MPIDI_CH4R_get_mapsize```
Both (1) and (2) are always inlined (even with noinline option).

In this PR, the `(1) CH4R_win_allocate_*` is directly called by `MPID_win_allocate_*`. However, we do not include the corresponding header files at ch4 layer...That is why I have to include it here.

Please note that this is just a temporary workaround. We eventually want to move all the `CH4R_*` routines into `ch4/generic`, and have a single header file that defines all the routines' signature. After talked to @raffenet , we want to just do the refactoring during hackathon and use the temporary workaround in this PR.